opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -409,13 +426,13 @@ namespace Opm {
                 if (elem.partitionType() != Dune::InteriorEntity)
                     continue;
 
-		unsigned globalElemIdx = elemMapper.index(elem);
+        unsigned globalElemIdx = elemMapper.index(elem);
```

Comment:
Anyone has some idea about what is going on about these indentations here?  TAB or Space related issue?

Reply:
@GitPaean It seems like it. My apologies. Seems like one alignment out (4 space) tab.

Reply:
Please recover the unnecessary changes in BlackoilModelEbos.hpp, like these indentation and also the above `const double dt = timer.currentStepLength();`. Basically, any change should have a good reason hopefully.